src/components/Counter/Counter.st.css

```diff
@@ -43,34 +49,47 @@
     background: color(value(DefaultBackgroundColor));
     border-width: 1px;
     border-style: solid;
-    border-color: color(value(counterColor));
+    border-color: applyOpacity(value(counterColor), 0.6);
 }
 
 .root:error {
     border-color: value(DefaultErrorColor);
 }
 
+.root:hover:not(:disabled):not(:error),
+.root:focus:not(:disabled):not(:error){
+    border-color: value(counterColor);
+}
+
 .root:disabled {
     border-color: color(value(counterDisabledColor));
 }
 
 .btn {
+    -st-extends: TPAIconButton;
     background: none;
     padding: 6px;
     border: none;
     cursor: pointer;
 }
 
-.btn path {
+.root .btn::icon svg path {
     fill: color(value(counterColor));
+    stroke: none;
 }
```

Comment:
Note that you should also be able write it like this:
```suggestion
.btn {
    -st-extends: TPAIconButton;
    -st-mixin: TPAIconButton(
        IconColor color(value(counterColor))
    );
    background: none;
    padding: 6px;
    border: none;
    cursor: pointer;
}
```
similar to what I did in the Tabs component:
https://github.com/wix/wix-ui-tpa/blob/073e8d562ad2f4cc13426ac30d58d1e17188001b/src/components/Tabs/Tabs.st.css#L102